.readthedocs.yml

```diff
@@ -8,9 +8,7 @@ sphinx:
   # This can be re-enable once RTD is updated with https://github.com/readthedocs/readthedocs.org/issues/7116
   fail_on_warning: False
 
-formats:
-  - pdf
-  - epub
+formats: all
```

Comment:
Maybe add a comment that we have "all" as it should include htmlzip and we want htmlzip for offline docs readers.

Just a help with someone touching this in the future.
Most probably that person will not check the whole history of this line..
and might think that "all" is excessive and html and PDF should be enough.